src/battle_ai_util.c

```diff
@@ -799,9 +796,9 @@ s32 AI_CalcDamage(u16 move, u8 battlerAtk, u8 battlerDef)
             dmg = 0;
         // Two hits of Super Fang halves HP twice, leaving target with 25% HP
         else if (AI_DATA->atkAbility == ABILITY_PARENTAL_BOND)
-            dmg = gBattleMons[battlerDef].hp * 3 / 4;
+            dmg = max(1, gBattleMons[battlerDef].hp * 3 / 4);
```

Comment:
Should be 2, if we hit twice then we have a minimum of 2 damage as it's 1 damage per hit.